internal/install/application_configuration.go

```diff
@@ -34,11 +34,28 @@ type stack struct {
 	ImageRefOverrides map[string]ImageRefs `yaml:"image_ref_overrides"`
 }
 
+func checkImageRefOverride(envVar string, fallback string) string {
+	refOverride := os.Getenv(envVar)
+	if refOverride == "" {
+		return fallback
+	}
+
+	return refOverride
+}
+
 func (s stack) ImageRefOverridesForVersion(version string) ImageRefs {
 	refs, ok := s.ImageRefOverrides[version]
+
+	kibanaRefOverride := checkImageRefOverride("KIBANA_IMAGE_REF_OVERRIDE", DefaultKibanaRef)
```

Comment:
I guess you don't have to define to `DefaultKibanaRef` as it's already there: https://github.com/elastic/elastic-package/blob/4c8b1fff98cc69a1814d7e40c27a0831c5a76647/internal/install/application_configuration_yml.go#L11